### PR TITLE
SSO

### PR DESCRIPTION
**Pull request by ArifPMan**
Updated SSO information in the docs. Please review and approve the changes..

### CODE REVIEW

File: v6/enterprise/sso/admin_sso.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ warning: false
 
 Only an Enterprise team administrator (admin) can configure single sign-on (SSO) for a Postman team.
 
-In [Team Settings](https://app.getpostman.com/dashboard/teams/edit){:target="_blank"} for an enterprise plan, scroll down and click the **Add a new authentication method** button.
+Go to [Team Settings](https://app.getpostman.com/dashboard/teams/edit){:target="_blank"} for an enterprise plan. The Team Settings screen appears as below:
+
+[![sso enable](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/SSO_AdminEnable1.png)
+
+Even after configuring SSO in your environment, you can still control the SSO authentication system using Postman's Enable/Disable option in Settings. Click the orange circled button to turn on/off your SSO authetication system. For instance, this option might be handy to an admin when an employee leaves the company as the authentication control rests with the admin.
```
Comment (numaanashraf) [inline v6/enterprise/sso/admin_sso.md]: typo `authentication`

Comment (numaanashraf) [inline v6/enterprise/sso/admin_sso.md]: > For instance, this option might be handy to an admin when an employee leaves the company as the authentication control rests with the admin.

Not sure if admins would enable/disable the SSO when an employee leaves. This option is team level and would disable/enable SSO for the whole team and not for a single team member.

Comment (numaanashraf) [inline v6/enterprise/sso/admin_sso.md]: > Even after configuring SSO in your environment, you can still control the SSO authentication system using Postman's Enable/Disable option in Settings.

After configuring an SSO authentication for your Postman team, you can use the toggle option in Settings (as illustrated above) to turn on/off this SSO authentication.

Comment (ArifPMan) [reply v6/enterprise/sso/admin_sso.md]: Fixed all issues as pointed...

**Pull request merged**